build/bin/vars

```diff
@@ -11,7 +11,11 @@ full_percona_dbaas_cli_version=${percona_dbaas_cli_version}-$(git rev-parse --sh
 echo -e "\n\n\n>>> full_percona_dbaas_cli_version=${full_percona_dbaas_cli_version} \n\n\n"
 
 dbaas_version=${percona_dbaas_cli_version}
-dbaas_release=1.$(git rev-parse --short HEAD)
+if [[ "$RELEASE_BUILD" == 'NO' ]]; then
+    dbaas_release=0.$(date -u +"%y%m%d%H%M").$(git rev-parse --short HEAD)
```

Comment:
$(git rev-parse --short HEAD)
If it does not work for deb I can remove it. (I think it will be nice to have commit id for debug for autobuilds,  but  it is up to you  )